### PR TITLE
Allocation domain `SchedulerRuntimeInfo` fix

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
!build

File: test/test_allocation_domain.cpp [hunk 1]
```diff
@@ -888,4 +889,121 @@ TEST_F(AllocationDomainTest, NHWC2d_To_NHWC2d_cacheAfter_CUDA) {
...
+    auto options = at::TensorOptions().dtype(at::kHalf).device(at::kCUDA, 0);
+    at::Tensor t0 = at::randn({n * h * w * c}, options)
+                        .as_strided({n, c, h, w}, {5, 1, 3, c});
+    auto vec = SchedulerRuntimeInfo(fusion_ptr.get(), {t0})
+                   .getMaxVectorizableWidth(tv0);
+    ASSERT_EQ(vec, 1);
+  }
+
+  // wc contiguous, good stride
+  {
```
Comment (naoyam) [inline test/test_allocation_domain.cpp:954]: Is the generated code vectorized with a factor of 2?

Comment (zasdfgbnm) [reply test/test_allocation_domain.cpp:954]: No, this fusion is not scheduled. `getMaxVectorizableWidth` is mostly used to compute heuristics and determine the max safe vectorization factor. It's the scheduler's decision to vectorize or not.

Comment (naoyam) [reply test/test_allocation_domain.cpp:954]: OK, that was my guess as I haven't seen any update about the scheduler.

**Review by naoyam — APPROVED** [review]
LGTM

File: test/test_allocation_domain.cpp (hunk 1 above)
Comment (zasdfgbnm) [reply test/test_allocation_domain.cpp:954]: Schedulers needs a lot more work to do, and I will likely do these work in a bunch of PRs.

**Pull request merged**